### PR TITLE
Make disasm printing more consistent and concise

### PR DESCRIPTION
**Pull request by Grazfather**

### CODE REVIEW

File: gef.py [hunk 1]
```diff
@@ -4933,25 +4933,25 @@ def context_code(self):
...
                         else:
-                            line+= Color.colorify("\tBranch NOT taken %s" % reason, attrs="bold red")
+                            line+= Color.colorify("\tNOT taken %s" % reason, attrs="bold red")
 
                 else:
-                    line+= "%#x\t %s" % (addr, content)
+                    line+= "%#x\t%s" % (addr, content)
 
-                print(line)
+                print(u"".join(line))
```
Comment (Grazfather) [inline gef.py]: Very minor, but the reason for this is that doing `str1 + str2` causes _3_ allocations, so this is slightly more efficient

Comment (hugsy) [inline gef.py]: This is not more efficient: see https://ideone.com/4VUZ0w
The string version is **always** faster.

**Comment by hugsy** [comment]
I reject this PR since some of the changes (and others) are already implemented in an unpushed version.

File: gef.py (hunk 1 above)
Comment (Grazfather) [inline gef.py]: fwiw, your test is only doing one append and one join, but the join version also has to allocate a dictionary for each call, so I don't think it's a very fair test. Here's a more realistic one: https://ideone.com/CLcSXD

**Comment by Grazfather** [comment]
35753086 implements most of this.

File: gef.py (hunk 1 above)
Comment (hugsy) [inline gef.py]: This is very interesting!

You're right, my test was incomplete. Using yours, I noticed that the array version is **always** faster on Python2 (by ~16%), whereas the string and table version are approx. the same speed on Python3. 

So you're right, on average it is a bit faster to use the table approach. I will integrate this.

Thank you.

Comment (Grazfather) [inline gef.py]: No problem!

**Pull request closed without merging**